addons/website_event/static/src/js/website_event.js

```diff
@@ -54,6 +54,10 @@ var EventRegistrationForm = Widget.extend({
             return ajax.jsonRpc($form.attr('action'), 'call', post).then(function (modal) {
                 var $modal = $(modal);
                 $modal.modal({backdrop: 'static', keyboard: false});
+                var $firstRow = $($modal.find('.row')[0]);
+                $($firstRow.find('input[type="text"]')[0]).val($modal.find('input[name="user_name"]').val());
+                $($firstRow.find('input[type="email"]')[0]).val($modal.find('input[name="user_email"]').val());
+                $($firstRow.find('input[type="tel"]')[0]).val($modal.find('input[name="user_phone"]').val());
```

Comment:
Since there is only one `input[type="email"]` and one `input[type="tel"]` couldn't you write `$firstRow.find('input[type="email"]').val($modal.find('input[name="user_email"]').val())` instead ?